test/zone_test.rb

```diff
@@ -59,6 +59,58 @@ class ZoneTest < DNS::Zone::TestCase
 mail IN A 78.47.253.85
 EOL
 
+  # zone file with multiple zones
+  ZONE_FILE_MULTIPLE_EXAMPLE =<<-EOL
+$ORIGIN lividpenguin.com.
+$TTL 3d
+@           IN  SOA  ns0.lividpenguin.com. luke.lividpenguin.com. (
+                           2013101406 ; zone serial number
+                           12h        ; refresh ttl
+                           15m        ; retry ttl
+                           3w         ; expiry ttl
+                           3h         ; minimum ttl
+                         )
+
+; a more difficult ; comment ( that is trying to break things!
```

Comment:
Can you make the test data focus on the multiple ORIGIN directives, these types of tests are already covered :)